client/reader/stream/post-lifecycle.jsx

```diff
@@ -11,8 +11,43 @@ import PostStore from 'lib/feed-post-store';
 import PostStoreActions from 'lib/feed-post-store/actions';
 import PostPlaceholder from './post-placeholder';
 import PostUnavailable from './post-unavailable';
+import ListGap from 'reader/list-gap';
 import CrossPost from './x-post';
-import XPostHelper from 'reader/xpost-helper';
+import RecommendedPosts from './recommended-posts';
+import XPostHelper, { isXPost } from 'reader/xpost-helper';
+import PostBlocked from 'blocks/reader-post-card/blocked';
+import Post from './post';
+import { RelatedPostCard } from 'blocks/reader-related-card-v2';
+import { recordTrackForPost, recordAction } from 'reader/stats';
+import {
+	EMPTY_SEARCH_RECOMMENDATIONS,
+	IN_STREAM_RECOMMENDATION,
+} from 'reader/follow-button/follow-sources';
+
+function EmptySearchRecommendedPosts( { post } ) {
```

Comment:
I wonder if its possible to somehow combine this with our other type of RecommendedPost